### DISCUSSION TITLE
Cannot change the player option

### DISCUSSION THREAD

**Discussion by i-tsx** [Androz2091/discord-player#746] [Q&A]
- Node Version: latest
 - Discord Player Version: latest
 - Discord.js Version: latest

I can't change the player options, there is no error on the console

![image](https://user-images.githubusercontent.com/69731556/131228200-0bb04af9-20f2-448d-847f-bdc21953820f.png)

![image](https://user-images.githubusercontent.com/69731556/131228210-b8f72069-c119-4d60-9ad7-707ec0149567.png)

**Comment by twlite** [comment]
Hello, all the top level player options are no longer supported in v5 and you need to specify it while using `createQueue`

**Comment by twlite** [comment]
something like

```js
const queue = player.createQueue(interaction.guild, {
    metadata: interaction.channel,
    autoSelfDeaf: false
});
```

**Comment by twlite** [comment]
is this resolved @abdusyr? I am moving this issue to discussions tab
